pyat/at/load/madx.py

```diff
@@ -707,13 +709,18 @@ def beta() -> float:
 
             params["_length"] = cell_length
             rev = beta() * clight / cell_length
+
+            # Set the frequency of cavities in which it is not specified
+            hn = 2147483647
```

Comment:
Where does number come from?

Reply:
2^31 - 1
In python, here is no "maxint", the `int` type has no upper bound. I took the largest positive integer that can be coded in a 32bit integer… Any large value fits…